geopandas/tests/test_geocode.py

```diff
@@ -13,16 +14,12 @@
 from pandas.util.testing import assert_series_equal
 from geopandas.tests.util import mock, assert_geoseries_equal
 
+geopy = pytest.importorskip("geopy")
 
-def _skip_if_no_geopy():
-    try:
-        import geopy
-    except ImportError:
-        raise pytest.skip("Geopy not installed. Skipping tests.")
-    except SyntaxError:
-        raise pytest.skip("Geopy is known to be broken on Python 3.2. "
-                          "Skipping tests.")
 
+@pytest.mark.skipif(
+    sys.version_info[0], sys.version_info[1]== (3,2),
+    reason="Geopy is known to be broken on Python 3.2.")
```

Comment:
It seems you didn't commit the removal of this one? (as in the comment you said you did?)

Reply:
No idea what happened there, should be fixed now!